src/core/simulation_results.jl

```diff
@@ -1,9 +1,6 @@
-const KNOWN_PATHS =
-    ["data_store", "logs", "models_json", "recorder", "results", "simulation_files"]
-
 function check_folder_integrity(folder::String)
     folder_files = readdir(folder)
-    alien_files = [f for f in folder_files if f ∉ KNOWN_PATHS]
+    alien_files = [f for f in folder_files if f ∉ KNOWN_SIMULATION_PATHS]
```

Comment:
```suggestion
    alien_files = filter(!∈(KNOWN_SIMULATION_PATHS), folder_files)
```

I believe in Julia 1.6 [1] you'll be able to do `filter(∉(KNOWN_SIMULATION_PATHS), folder_files)`. 

[1] https://github.com/JuliaLang/julia/pull/38475